src/main/java/practice/StreamPractice.java

```diff
@@ -35,7 +55,11 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * Example: select men who can be recruited to army (from 18 to 27 years old inclusively).
      */
     public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(p -> p.getAge() >= fromAge
+                        && p.getAge() <= toAge
+                        && p.getSex() == Person.Sex.MAN)
```

Comment:
```suggestion
                .filter(p -> p.getAge() >= fromAge
                        && p.getAge() <= toAge
                        && p.getSex() == Person.Sex.MAN)
```
Неймінг змінних
If you have a long boolean expression you may create your own implementation of Predicate and use it in your filter - це з рекомендацій до завдання

Reply:
here, I will discuss with Max to compere this and Perducator

Reply:
changed naming